main.go

```diff
@@ -2,17 +2,20 @@ package main
 
 import (
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io/ioutil"
 	"net/http/httputil"
 	"os"
 	"os/exec"
+	"reflect"
 	"regexp"
 	"sort"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/ghodss/yaml"
```

Comment:
Because that's what was used in vault/secrets.go, added there by @jhunt on 2016/02/09 to handle the exact same thing that we're doing here, so I knew how it worked and that it worked and the output would be consistent with the other output.